lib/base-controller.js

```diff
@@ -3,10 +3,12 @@
 var Controller = require('hmpo-form-wizard').Controller;
 var util = require('util');
 var _ = require('lodash');
+var mixins = require('./middleware/mixins');
 
 var BaseController = function BaseController(options) {
   this.confirmStep = options.confirmStep || '/confirm';
   Controller.apply(this, arguments);
+  this.use(mixins);
```

Comment:
By being middleware, the function is called for every type of request, it's not limited to a particular `req.method`. I think it is only required for `get` requests, so perhaps calling it from `getValues`, or `get` or something similar. I think there is a `render` defined in HMPO form controller that might be suitable for overriding in HOF base controller??